cardano-chain-gen/test/testfiles/config-conway-hf-epoch1/test-db-sync-config.json

```diff
@@ -1,6 +1,7 @@
 {
   "EnableLogMetrics": false,
   "EnableLogging": true,
+  "EnableFutureGenesis" : true,
```

Comment:
I'm going to wait to review this PR until https://github.com/IntersectMBO/cardano-db-sync/pull/1737/files is merged as it seems there is new duplicate code in both PRs